internal/querynode/collection_replica.go

```diff
@@ -296,6 +296,19 @@ func (colReplica *collectionReplica) getPartitionIDs(collectionID UniqueID) ([]U
 	return collection.partitionIDs, nil
 }
 
+func (colReplica *collectionReplica) getIndexedFieldIDByCollectionIDPrivate(collectionID UniqueID) ([]FieldID, error) {
```

Comment:
It seems that returned fields do not all have index

Reply:
Yes, so after call this we'll check if the field has been indexed using `hasLoadIndexForIndexedField`.